recipes/_packages.rb

```diff
@@ -0,0 +1,20 @@
+include_recipe 'apt'
+
+type = 'deb'
+case node['platform_family']
+when 'fedora', 'rhel'
+  type = 'rpm'
+end
+
+packagecloud_repo "StackStorm/stable" do
+  type type
+end
+
+directory ":create #{node['openstack-mistral']['home']}" do
```

Comment:
The directory `/opt/stackstorm/mistral` [is hardcoded](https://github.com/StackStorm/st2-packages/blob/master/packages/st2mistral/rpm/st2mistral.spec#L59) in `st2mistral` `deb/rpm` packages and cannot be changed.

I think we can remove this, as well as entire `['openstack-mistral']['repo']['home']` setting.